ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -77,41 +78,57 @@ export class Pay extends IronfishCommand {
       this.exit(1)
     }
 
-    if (amount == null || Number.isNaN(amount)) {
-      const response = await client.getAccountBalance({ account: from })
+    if (fee == null || Number.isNaN(fee)) {
+      let dynamicFee: string | null
+
+      try {
+        // fees p25 of last 100 blocks
+        dynamicFee = displayIronAmount(
+          oreToIron((await client.getFees({ numOfBlocks: 100 })).content.p25),
+        )
+      } catch {
+        dynamicFee = null
+      }
 
       const input = Number(
         await CliUx.ux.prompt(
-          `Enter the amount in $IRON (balance available: ${displayIronAmountWithCurrency(
-            oreToIron(Number(response.content.confirmed)),
-            false,
-          )})`,
+          `Enter the fee amount in $IRON (min: 0.00000001${
```

Comment:
Any particular reason you're switching order here? I can imagine this may break some scripts.

Reply:
i think its useful to have the `max` amount payable in the 'amount' prompt and that has to come after the fee has been inputted. just easier UX for users to not have to do the extra mental math.

Reply:
I see the argument here, but I think I would prefer to leave as is. Changing the order of params will likely break some scripts/workflows and not sure it's worth it.